src/backend/distributed/planner/relation_restriction_equivalence.c

```diff
@@ -2023,25 +2022,19 @@ RemoveDuplicateJoinRestrictions(JoinRestrictionContext *joinRestrictionContext)
 
 
 /*
- * JoinRestrictionListExistsInContext returns true if the given joinRestrictionInput
+ * ShouldAddJoinRestrictionToContext returns true if the given joinRestriction
```

Comment:
Isn't this name the opposite of what it does?   If `ShouldAddJoinRestrictionToContext ` returns true, I'd expect us add to the list. But we skip?

I  guess we can use `ContextCoversJoinRestriction(joinRestrictionContext, joinRestriction)` 

(so, switch the order of the parameters as well)

Reply:
>Isn't this name the opposite of what it does? 
yeah, that's true